plugins/inventory/proxmox.py

```diff
@@ -27,17 +27,29 @@
         choices: ['community.general.proxmox']
         type: str
       url:
-        description: URL to Proxmox cluster.
+        description:
+          - URL to Proxmox cluster.
+          - If the value is not specified in the task, the value of environment variable C(PROXMOX_URL) will be used instead.
         default: 'http://localhost:8006'
         type: str
+        env:
+          - name: PROXMOX_URL
```

Comment:
```suggestion
          - name: PROXMOX_URL
            version_added: 2.2.0
```
(same for the others).

Reply:
Addressed in https://github.com/ansible-collections/community.general/pull/1645/commits/a111f6a34146162349b691bfda9269e30ee660bd

Reply:
@felixfontein Shouldn't this be `2.0.0` instead of `2.2.0`?

Reply:
@Akasurde thanks, yes, definitely! That's a really stupid typo by me...